src/site/content/en/blog/web-platform-01-2022/index.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "New to the web platform in January"
+subhead: >
+  In this post discover some of the interesting features that have landed in stable and beta web browsers during January 2022. 
+description: >
+  In this post discover some of the interesting features that have landed in stable and beta web browsers during January 2022.  
+date: 2022-02-01
+hero: image/kheDArv5csY6rvQUJDbWRscckLr1/8j6GZWWy9wpuQfTOkrJy.png
+alt: Balloons launching.
+authors:
+  - rachelandrew
+tags:
+  - blog
+---
+
+## Stable browser releases
+
+In January Chrome 97, and Firefox 96 became stable.
+
+Firefox 96 ships the color function [`hwb()`](https://developer.mozilla.org/docs/Web/CSS/color_value/hwb()). This function expresses a color by hue, lightness, and blackness.
+
+{% BrowserCompat 'css.types.color.hwb' %}
+
+Also in Firefox 96 is the CSS [`color-scheme`](https://developer.mozilla.org/docs/Web/CSS/color-scheme) property. This property gives you a way to indicate the color schemes an element can be rendered in. For example, to indicate that `.widget` can be rendered in the operating system’s light or dark modes, use the following CSS.
+
+```css
+.widget {
+  color-scheme: light dark;
+}
+```
+
+With color-scheme landing in Firefox, this means the property is available in Chrome, Firefox, and Safari.
+
+{% BrowserCompat 'css.properties.color-scheme' %}
+
+## Beta browser releases
+
+Beta browser versions give you a preview of things that will be in the next stable version of the browser. It’s a great time to test new features, or removals, that could impact your site before the world gets that release.
+
+Betas landing in January were [Chrome 98](https://blog.chromium.org/2022/01/chrome-98-beta-color-gradient-vector.html), [Firefox 97](https://www.mozilla.org/en-US/firefox/97.0beta/releasenotes/), and Safari 15.4 Beta 1.
+
+Chrome 98 supports COLRv1 color gradient vector fonts as an additional new font format. A color font contains glyphs with multiple colors in them, which can be for example an emoji or a country flag or a multi-colored letter.
+
+Read more about this new font format in [COLRv1 Color Gradient Vector Fonts in Chrome 98](https://developer.chrome.com/blog/colrv1-fonts/).
+
+Also included is the [structuredClone()](/structured-clone/) method for making deep copies of objects, and the CSS media queries `dynamic-range` and `video-dynamic-range`. There’s lots more that you can find by reading the [Chrome 98 beta post](https://blog.chromium.org/2022/01/chrome-98-beta-color-gradient-vector.html).
+
+[Chrome 98 DevTools](https://developer.chrome.com/blog/new-in-devtools-98/) brings you a whole raft of new features including the new [Full Accessibility Tree](https://developer.chrome.com/blog/full-accessibility-tree/).
```

Comment:
Is "a whole raft" a common UK phrase? I'm not familiar with it. It would probably be more localization-friendly to update to "several new features", but I also recognize this is a blog post which can benefit from specific voices.

Reply:
I don't think it's an uncommon phrase, but maybe it is, I'll change it!